src/pages/installcustom.rs

```diff
@@ -63,40 +63,27 @@ impl SimpleComponent for InstallCustomPage {
                 set_vexpand: true,
                 set_hexpand: true,
 
-                gtk::ScrolledWindow {
-                    #[local_ref]
-                    mounts -> gtk::Box {
-                        set_margin_horizontal: 16,
-                        set_spacing: 8,
-                        set_orientation: gtk::Orientation::Vertical,
-                        add_css_class: "content-list",
-                        set_vexpand: true,
-                        set_hexpand: true,
-                        set_valign: gtk::Align::Center,
-                        set_halign: gtk::Align::Fill,
-                    }
-                },
-
-                // FIXME: help me position this button!!!!
-
-                libhelium::OverlayButton {
-                    set_valign: gtk::Align::End,
-                    set_halign: gtk::Align::End,
-
-                    set_typeb: libhelium::OverlayButtonTypeButton::Primary,
-                    set_icon: "go-next",
-                    connect_clicked => InstallCustomPageMsg::Navigate(NavigationAction::GoTo(crate::Page::Confirmation)),
-                },
-
+                #[local_ref]
+                scrolled_window -> gtk::ScrolledWindow {},
+                append = &scrolled_window,
 
+                #[local_ref]
+                bottom_bar -> libhelium::BottomBar {
+                    set_mode: libhelium::BottomBarMode::Floating,
 
-                libhelium::BottomBar {
                     #[watch]
                     set_title: &t!("page-installcustom-title"),
 
                     #[watch]
                     set_description: &t!("page-installcustom-desc", num = model.choose_mount_factory.len()),
 
+                    #[local_ref]
+                    next_button -> libhelium::OverlayButton {
```

Comment:
```suggestion
                    #[name(next_button)]
                    libhelium::OverlayButton {
```
same for other places